getting_started/install_openshift.adoc

```diff
@@ -162,6 +162,12 @@ xref:../install/example_inventories.adoc#install-config-example-inventories[Exam
 Inventory Files] and select the example that most closely matches your desired cluster
 configuration.
 +
+
+[IMPORTANT]
+====
+Running Ansible playbooks with `--tags` or `--check` is not supported by Red Hat.
```

Comment:
s/with \`--tags\` or \`--check\`/with the \`--tags\` or \`--check\` options

Reply:
Agreed! Fixed and squashed. Thanks!